cmd/config.sample.toml

```diff
@@ -1,3 +1,9 @@
+[app]
+# log level is one of : INFO, DEBUG, ERROR
+log_level = "INFO"
+# maximum time (in seconds) allowed for a job to run once it has started execution
+job_ttl = "60s"
```

Comment:
This should be `default_job_ttl` to be clearer as it can be changed per job.

Reply:
Correct. Also, remove the `(in seconds)` bit from the comment since this is a `time.Duration`